cli/organizationmembers.go

```diff
@@ -11,16 +12,74 @@ import (
 )
 
 func (r *RootCmd) organizationMembers() *serpent.Command {
+	cmd := &serpent.Command{
+		Use:     "members",
```

Comment:
Is this ready for end users to start consuming? If not, do we want to start with it hidden?

Reply:
It is not, but the parent command is hidden:

https://github.com/coder/coder/blob/a8fe429b3411e0b69db793d4426e6e5d51501ced/cli/organization.go#L24-L24

So this is also hidden

Reply:
Nice 👍